Monika After Story/game/script-story-events.rpy

```diff
@@ -269,6 +269,198 @@ label monika_changename:
             m 1eua "Just let me know if you had a change of heart, ok?"
     return
 
+default persistent._mas_player_bday = None
+# check to see if we've already confirmed birthday in any way
+default persistent._mas_player_confirmed_bday = False
+# so we can still set the old_bday vars if she had your bday correct
+default persistent._mas_player_bday_correct = False
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="mas_birthdate",conditional="persistent.sessions['first_session'] is not None and datetime.date.today()>persistent.sessions.get('first_session',None).date() and not persistent._mas_player_confirmed_bday",action=EV_ACT_QUEUE))
```

Comment:
I've added a new function `mas_getFirstSesh`. Use that to get the first session to compare stuff. It will always return a datetime so you dont have to worry about None checking.